templates/search/results-fr.html

```diff
@@ -22,66 +22,204 @@
 </form>
 
 <section class="results">
-	<h2 class="wb-inv">Résultats de recherche</h2>
-	<p class="h4">[nombre de] résultats de recherché pour "[le terme recherché]"</p>
+	<div id="query-summary">
+		<h2 class="h4">[nombre de] résultats de recherché pour "[le terme recherché]"</h2>
```

Comment:
Please remove the `h4` class since the font size for this `h2` is defined through the CSS and must be consistent.